### PR TITLE
WinUI 890011 - Updated the API links for Scheduler tooltip content

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have updated the API links for the Scheduler tooltip.

### CODE REVIEW

File: winui/Scheduler/Appointments.md [hunk 1]
```diff
@@ -812,7 +812,7 @@ The following code example shows the usage of DataTemplate.
 </scheduler:SfScheduler>
 
 N>
-* This property will only be applicable when `EnableTooltip` is set to true.
+* This property will only be applicable when [EnableToolTip](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) is set to true.
```
Comment (Karthickmani97) [inline winui/Scheduler/Appointments.md:815]: Add API link for `CellRightPadding` in days, month view and `CellBottomPadding` in timeline view as well.

**Review by Karthickmani97 — COMMENTED** [review]
@TamilarasanGunasekaran , address amy review corrections.

**Review by Karthickmani97 — COMMENTED** [review]
@TamilarasanGunasekaran , Address my review corrections,

File: winui/Scheduler/Appointments.md [hunk 2]
```diff
@@ -762,15 +762,15 @@ An interactive tooltip provides additional details about the appointments on hov
 
 ### Enable tooltip for appointments
 
-To enable tooltip for the scheduler appointments, use the `EnableTooltip` property of `SfScheduler.` By default, `EnableTooltip` is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
+To enable tooltip for the scheduler appointments, use the [EnableToolTip](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) property of [SfScheduler](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html). By default, [EnableToolTip](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html#Syncfusion_UI_Xaml_Scheduler_SfScheduler_EnableToolTip) is set to `false.` To provide users with additional information or context about appointments, simply set this property to true.
```
Comment (Karthickmani97) [inline winui/Scheduler/Appointments.md:765]: Update same review correction here as well - https://github.com/syncfusion-content/winui-docs/pull/845

Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Appointments.md:765]: Updated.

File: winui/Scheduler/Day-Week-Views.md [hunk 3]
```diff
@@ -499,7 +499,7 @@ Customize the default appearance of view header by setting the [ViewHeaderTempla
 
 ## Cell right padding
 
-To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html#Syncfusion_UI_Xaml_Scheduler_DaysViewSettings_CellRightPadding) property of [DayViewSettings](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html) in the scheduler.
+To enhance interaction with a scheduler's appointments, you can customize the spacing between appointments and the right border of their cells using the [CellRightPadding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html#Syncfusion_UI_Xaml_Scheduler_DaysViewSettings_CellRightPadding) property of [DayViewSettings](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Scheduler.DaysViewSettings.html) in the [SfScheduler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Scheduler.SfScheduler.html).
```
Comment (Karthickmani97) [inline winui/Scheduler/Day-Week-Views.md:502]: Update WinUI API lin, remove WPF

Comment (TamilarasanGunasekaran) [reply winui/Scheduler/Day-Week-Views.md]: Updated.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**